src/fides/api/models/privacy_notice.py

```diff
@@ -27,6 +27,7 @@ class UserConsentPreference(Enum):
     opt_in = "opt_in"  # The user wants to opt in to the notice
     opt_out = "opt_out"  # The user wants to opt out of the notice
     acknowledge = "acknowledge"  # The user has acknowledged this notice
+    tcf = "tcf"  # Overall preference set for TCF where there are numerous preferences under the single notice
```

Comment:
Now that we capture the save of historical TCF preferences as a single row, instead of one row per TCF attribute, there is no overall "preference" for a TCF record. Neville recommended adding "tcf" so if someone queried this column for the preferences, we'd have no null records - 

To see more detailed preferences you'd look at `PrivacyPreferenceHistoryV2.tcf_preferences` jsonb column